1회차/이순철/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,12 @@
 import sys
 
 sys.stdin = open("_모음이보이지않는사람.txt")
+
+t = int(input())
+for t_case in range(1, t + 1):
+
+    word = input()
+    for i in word:
+        if i in 'aeiou':
+            word = word.replace(i, '')
+    print(f'#{t_case} {word}')
```

Comment:
깔끔하네요! 😄 
`replace` 함수를 아래와 같이 사용할 수도 있어요.
```suggestion
    for i in  'aeiou':
        word = word.replace(i, '')
    print(f'#{t_case} {word}')
```